Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Here the version of the registry is specified this storage branch uses.
 # It should always be a specific version to make sure builds are reproducible.
-ARG PACKAGE_REGISTRY=v0.6.0
+ARG PACKAGE_REGISTRY=v0.7.1
```

Comment:
@ruflin @jonathan-buttner would be good to know if there was a reason that prod was still on v0.6.0 and not v0.7.0

Reply:
Yeah I'm not sure about that. I see there were PRs to snapshot and staging though 🤔

Reply:
Yes, staging and snapshot were already on 0.7.0 ...

Reply:
The only reason what  not to include a breaking change in production.

Reply:
@ph What is this breaking change?